.ci/install_kata_image.sh

```diff
@@ -58,9 +58,9 @@ build_rust_image() {
 			;;
 	esac
 	info "Install ${target_image} to ${image_path}"
-	if [ -f "${image_path}" ]; then
+	local file="$(realpath ${image_path}/$(basename ${file_to_install}))"
+	if [ -f "${image_path}"  ] || [ -f "${file}" ]; then
```

Comment:
thanks - I think that `if [ -f "${file}" ]; then` should be enough

Reply:
@marcel-apf `$image_path` is a directory, we should ignore it, right? wdyt?

Reply:
@devimc sure, I'll respin soon(ish)

Reply:
@devimc done